### ISSUE TITLE
About shape file to geojson

### ISSUE THREAD

**Issue by iknow34languages** [evansiroky/timezone-boundary-builder#44]
i want to find a way to produce the geojson file from your shapefile.

but i find that ,the tool from the web or ogr2ogr can not produce the same content compared with your shapfile.geojson.zip  released on the web.
EG:
The geojson i produced (the same shapefile) : 
{ "tzid": "Africa\/Abidjan" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ -5.440683, 4.896553 ], [ -5.555659, 4.882248 ]...
-- WGS84 

different with your data
{"tzid":"Africa/Abidjan"},"geometry":{"type":"Polygon","coordinates":[[[-5.440683,4.896553],[-5.303699,4.912035]

Can you tell me which way you used to produce the geojson file

Thanks.

**Comment by evansiroky** [comment]
It could be the case that the coordinates of the linear ring are not being outputted in according to the right-hand rule.  Even if this happens, the geojson should be able to be parsed and should effectively represent the same geometry.  The geojson file in this project is produced using [jsts's GeoJSONWriter](https://bjornharrtell.github.io/jsts/doc/api/jsts.io.GeoJSONWriter.html).

**Comment by iknow34languages** [comment]
@evansiroky i try to use this to produce Time zone geojson Thanks!

**Comment by iknow34languages** [comment]
@evansiroky i am sorry to tell that i tried a many ways to change time shape file to geojosn file you released;  but it still different with your version.
the way "jsts's GeoJSONWriter  ," i can not find a good example for me to start.
i am a newer for java script.

**Comment by evansiroky** [comment]
@iknow34languages Can you please show me a picture of a map showing the differences?  Based on your current input I don't have enough information to resolve this issue.

**Comment by iknow34languages** [comment]
@evansiroky i put the raw position data in github:https://github.com/iknow34languages/Problems/tree/questions/question_1
include my own data from QGIS ,and the data from your combined.json

Thanks

**Comment by evansiroky** [comment]
The data in those files do not contain valid geojson and therefore cannot possibly be directly obtained from the output of this project.  However, if I assume that the contents of those files are merely a flattened array of what would be a polygon definition then they effectively represent the same exact thing.  After translating your data into valid GeoJson and rendering it in geojson.io I can see the exact same shape being rendered.

This is a screenshot of the first file translated to valid geojson on geojson.io:

<img width="357" alt="screen shot 2018-07-23 at 8 33 49 am" src="https://user-images.githubusercontent.com/3112493/43087192-1af1c21e-8e54-11e8-852c-c7192ae35ece.png">

And here is a screenshot of the second file translated to valid geojson on geojson.io:

<img width="377" alt="screen shot 2018-07-23 at 8 37 58 am" src="https://user-images.githubusercontent.com/3112493/43087227-2cb10852-8e54-11e8-837b-ce21d3fc6a53.png">

**Comment by iknow34languages** [comment]
@evansiroky yes,They are very similar from the QGIS tool  and the same shape as you show on map;

![image](https://user-images.githubusercontent.com/31202141/43114296-e3d99cba-8f30-11e8-8ae8-6995cf674dc6.png)


so you mean they should own the same accuracy Right?

Thanks!

**Comment by evansiroky** [comment]
They're not just similar.  They are exactly equivalent.

**Comment by iknow34languages** [comment]
@evansiroky ok ok ok
